ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -41,21 +44,44 @@ object Assertions {
     }
   }
 
+  /** non-regex overload for assertGrpcError which just does a substring check.
+    */
+  def assertGrpcError(
+      t: Throwable,
+      expectedCode: Status.Code,
+      exceptionMessageSubString: String,
+      checkDefiniteAnswerMetadata: Boolean = false,
```

Comment:
How about also allowing to check for the value itself?

Reply:
It's always `false` so far. However, I wouldn't like to hardcode the assertion to `false`, as Canton may want to set it to `true` somewhere at some point. Do you think it's ok to add `expectedDefiniteAnswerMetadata: Option[Boolean]` and assert it only if it's `Some`?

Reply:
Yes, that looks like a good solution to me.